### PR TITLE
TST: Refactor data loading script

### LINKED ISSUE #524: Resolve clang build issues on Jenkins
Ongoing in #522

### PR DESCRIPTION
**Pull request by laserson**
Closes #524

### CODE REVIEW

File: ibis/tests/util.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def __init__(self):
                                       '/tmp/__ibis_test')
         self.test_data_db = os.environ.get('IBIS_TEST_DATA_DB', 'ibis_testing')
         self.test_data_dir = os.environ.get('IBIS_TEST_DATA_HDFS_DIR',
-                                            '/__ibis/ibis-testing-data')
+                                            '/tmp/__ibis_test_data')
```
Comment (wesm) [inline ibis/tests/util.py]: is this necesary? Going to break my notebooks

Comment (laserson) [inline ibis/tests/util.py]: Not necessary, but was trying to make things more consistent.  Revert?

Comment (wesm) [inline ibis/tests/util.py]: Unless there's a need. We can always change later

Comment (laserson) [inline ibis/tests/util.py]: done.

**Comment by laserson** [comment]
@wesm this is ready for merge AFAICT.

**Comment by laserson** [comment]
This should make all the Jenkins jobs pass

**Comment by wesm** [comment]
looks great, thank you

**Pull request closed without merging**